pkg/auth/providers/saml/saml_client.go

```diff
@@ -316,8 +316,14 @@ func (s *Provider) FinalizeSamlLogout(w http.ResponseWriter, r *http.Request) {
 	if err != nil {
 		log.Debugf("SAML [FinalizeSamlLogout]: response validation failed: %v", err)
 		if parseErr, ok := err.(*saml.InvalidResponseError); ok {
-			log.Debugf("SAML RESPONSE: ===\n%s\n===\nSAML NOW: %s\nSAML ERROR: %s",
-				parseErr.Response, parseErr.Now, parseErr.PrivateErr)
+			// Note: If access to the response itself is needed (debugging)
+			// just add `parseErr.Response` to the log statement, like:
+			//
+			// log.Debugf("SAML RESPONSE: ===\n%s\n===\nSAML NOW: %s\nSAML ERROR: %s",
+			//	parseErr.Response, parseErr.Now, parseErr.PrivateErr)
```

Comment:
Is there any reason why we need this comment? It seems self-evident. If you need the response to be logged you add it to the statement.

Reply:
It is a bit of a reminder where to find the response to be added. The pre-made statement is then just convenience, i.e. change what is commented vs. not.

Reply:
Trimmed the comment now, see a7adf2e588d8e7ae10de89efb77837fcf0b9e5d9.
Please reopen if the entire comment should be removed.

Reply:
Ultimately it's up to you but IMHO there is no reason to have this comment at all.
But if you keep it can you please make sure it's not a free-floating (dangling) comment. In other words there shouldn't be any EOLs between the comment and the statement it refers to. Go doesn't like such comments very much.